packages/shaderlab/src/shaders/shadingPBR/LightIndirectPBR.glsl

```diff
@@ -11,7 +11,9 @@
 #ifndef FUNCTION_CLEAR_COAT_IBL
     #define FUNCTION_CLEAR_COAT_IBL evaluateClearCoatIBL
 #endif
-
+#ifndef FUNCTION_IRIDESCENCE_IBL
+    #define FUNCTION_IRIDESCENCE_IBL evaluateIridescenceIBL
+#endif
```

Comment:
may not export this interface, if user want change logic with thin, can just rewrite `FUNCTION_SPECULAR_LOBE`